packages/elements/src/clock/index.ts

```diff
@@ -220,27 +243,50 @@ export class Clock extends ResponsiveElement {
   /**
    * Display the digital clock in 12hr format.
    */
-  @property({ type: Boolean, attribute: 'am-pm' })
+  @property({ type: Boolean, attribute: 'am-pm', reflect: true })
   public amPm = false;
 
   /**
    * Display the seconds segment.
    */
-  @property({ type: Boolean, attribute: 'show-seconds' })
+  @property({ type: Boolean, attribute: 'show-seconds', reflect: true })
   public showSeconds = false;
 
-  /**
-   * Enabled interactive mode. Allowing the user to offset the value.
-   */
-  @property({ type: Boolean })
-  public interactive = false;
-
   /**
   * Display clock in analogue style.
   */
   @property({ type: Boolean, reflect: true })
   public analogue = false;
 
+  private _interactive = false;
+
+  /**
+   * Enable interactive mode. Allowing the user to offset the value.
+   * Set tabIndex >= 0 to make the clock interactive
+   * Set/remove tabIndex to move clock to non interactive mode
+   * @param value Set interactive mode
+   */
+  @property({ type: Boolean })
+  public set interactive (value: boolean) {
+    const newValue = !!value;
+    const oldValue = this.tick;
```

Comment:
`tick`?

Reply:
My mistake. It should be interactive. Updated to interactive.